Tests/SonarScanner.MSBuild.Common.Test/EnvironmentBasedPlatformHelperTests.cs

```diff
@@ -40,10 +40,4 @@ public void DirectoryExists_WithCurrentDirectory()
     {
         Instance.DirectoryExists(Environment.CurrentDirectory).Should().BeTrue();
     }
-
-    [TestMethod]
-    public void IsWindowsAndMacOSX_AreMutuallyExclusive()
```

Comment:
After removing this test method, do we have some tests for the behavior of the `OperatingSystem` property, that replaces `IsWindows` and `IsMacOSX`?

Reply:
I don't see how it can be tested with UTs. I marked it with the` [ExcludeFromCodeCoverage]` attribute. I'll add an explanation that it was tested manually.